sdk/core/azure-core-tracing-opencensus/src/main/java/com/azure/core/tracing/opencensus/OpenCensusTracer.java

```diff
@@ -242,10 +242,10 @@ private void addSpanRequestAttributes(Span span, Context context, String spanNam
         span.putAttribute(COMPONENT, AttributeValue.stringAttributeValue(parseComponentValue(spanName)));
         span.putAttribute(
             MESSAGE_BUS_DESTINATION,
-            AttributeValue.stringAttributeValue(getRequestKeyAttribute(context, ENTITY_PATH_KEY)));
+            AttributeValue.stringAttributeValue(getOrDefault(ENTITY_PATH_KEY, "", String.class, context)));
         span.putAttribute(
             PEER_ENDPOINT,
-            AttributeValue.stringAttributeValue(getRequestKeyAttribute(context, HOST_NAME_KEY)));
+            AttributeValue.stringAttributeValue(getOrDefault(HOST_NAME_KEY, "", String.class, context)));
```

Comment:
Thinking about the arguments... it should be (Context context, String key, T defaultValue, Class<T> clazz).